acceptance/lib/helper.rb

```diff
@@ -26,13 +26,13 @@ def self.initialize_config(options)
                          nil, "Puppet Version", "PUPPET_VERSION", nil) ||
                          get_puppet_version
 
-    # SERVER-339 - puppet-agent commit 49f7b1b967e44020587ab087f87e2f5e3ff2b1f6
-    # corresponds to packaged development version located at
-    # http://builds.delivery.puppetlabs.net/puppet-agent/49f7b1b967e44020587ab087f87e2f5e3ff2b1f6/
+    # SERVER-339, SERVER-386 - puppet-agent version corresponds to packaged
+    # development version located at http://builds.delivery.puppetlabs
+    # .net/puppet-agent/
     # TODO: This build version needs to be updated to a released version
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Development Build Version",
-                         "PUPPET_BUILD_VERSION", "49f7b1b967e44020587ab087f87e2f5e3ff2b1f6")
+                         "PUPPET_BUILD_VERSION", "bc00b6e7106846aae0af5b915976a8d192e53803")
```

Comment:
Not a direct comment on this PR, but this is kind of gross. Maybe down the road we can get this value from the .gitmodules file?